__tests__/pages/__snapshots__/dashboard.test.js.snap

```diff
@@ -1203,7 +1203,7 @@ exports[`Dashboard renders Dashboard 1`] = `
               class="font-display font-bold text-left pb-6"
             >
               <a
-                href="/dashboard"
+                href="https://www.canada.ca/en/services/benefits/ei/employment-insurance-reporting.html#Internet-Reporting-Service"
```

Comment:
This works for now, but from our first merge of dashboard --> dev, we were asked to make these with `next/link`. This is the snapshot, not sure what the actual file is using. Should be revisited